pom.xml

```diff
@@ -341,6 +341,49 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+
+                    <!--
+                      This insures that the windup-web-redirect directory exists.
+                      Without this, the CLI script may fail later on.
+                    -->
+                    <execution>
+                        <id>create-windup-web-redirect-directory</id>
+                        <phase>process-resources</phase>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${project.build.directory}/${wildfly.directory}/windup-web-redirect"/>
+                            </target>
+                        </configuration>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                    </execution>
+                    <!--
+                        This plugin causes a small sleep before packaging. This fixes a
```

Comment:
I believe this is why we are seeing intermittent failures on CI. I'm not 100% sure, though.